android/src/main/java/com/stellarscript/vlcvideo/VLCVideoProps.java

```diff
@@ -23,6 +23,11 @@
     static final int PAUSE_COMMAND_ID = 2;
     static final String SEEK_COMMAND_NAME = "seek";
     static final int SEEK_COMMAND_ID = 3;
+    static final String SET_SPU_COMMAND_NAME = "spu";
+    static final int SET_SPU_COMMAND_ID = 4;
+    static final String SET_AUDIO_COMMAND_NAME = "audio";
+    static final int SET_AUDIO_COMMAND_ID = 5;
     static final int SEEK_COMMAND_TIME_ARGUMENT_INDEX = 0;
```

Comment:
remove those 2 constants. no need of them. hardcode `0` where it is needed. its allways 0